src/allmydata/test/test_magic_folder.py

```diff
@@ -188,8 +188,8 @@ def test_load_legacy_no_dir(self):
 
         with self.assertRaises(Exception) as ctx:
             magic_folder.load_magic_folders(self.basedir)
-        self.assertIn(
-            "there is no directory at that location",
+        not self.assertIn(
+            "no directory could be created",
```

Comment:
should be self.assertNotIn probably

Reply:
Yes, right. Thank you for pointing it out.